wave_ebnf.md

```diff
@@ -62,7 +62,7 @@ record-fields ::= ws record-field ws
                 | record-fields ',' record-field
 record-field ::= label ws ':' ws value
 
-label ::= word
+label ::= '%'? word
        | label '-' word
```

Comment:
I think something like this would be more accurate:

```suggestion
label ::= '%'? inner-label
inner-label ::= word
              | inner-label '-' word
```

Reply:
Never mind, I just couldn't parse in my head!

Reply:
Oops, yeah, that looks right.

Reply:
I kind like the inner-label factoring in any case.